options.go

```diff
@@ -1083,6 +1083,18 @@ func Values(vals ...float64) interface {
 	return floatValues{vals}
 }
 
+type validateOpts struct {
+	errorHandler ErrorHandler
+}
+
+// ValidateOption is an option that can be passed to SpatialRef.Validate()
+//
+// Available ValidateOptions are:
+//  - ErrLogger
+type ValidateOption interface {
```

Comment:
Please rename this to `SpatialRefValidateOption` as Validate by itself is too generic